crypto/objects/objects.txt

```diff
@@ -933,11 +933,39 @@ nist_hashalgs 1		: SHA256		: sha256
 nist_hashalgs 2		: SHA384		: sha384
 nist_hashalgs 3		: SHA512		: sha512
 nist_hashalgs 4		: SHA224		: sha224
+nist_hashalgs 5		: id-sha512-224		: sha512-224
+nist_hashalgs 6		: id-sha512-256		: sha512-256
```

Comment:
"equivalent"? None of additions are *equivalent* of anything. sha512-\*, sha3-\* and shake\* are *related* *indirectly* to sha1 and sha2 by means of usage with dgst command, that's all.